modules/core/src/main/java/org/apache/ignite/configuration/PageReplacementMode.java

```diff
@@ -0,0 +1,85 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.apache.ignite.configuration;
+
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Defines memory page replacement algorithm. A mode is set for a specific {@link DataRegionConfiguration}.
+ */
+public enum PageReplacementMode {
+    /**
+     * Random-LRU algorithm.
+     *
+     * Every time page is accessed, its timestamp gets updated. When a page fault occurs and it's required to replace
+     * some pages, the algorithm randomly chooses 5 pages from the page memory and evicts a page with the latest
```

Comment:
seems hardcoded "5" need to be replaced with {@link RandomLruPageReplacementPolicy#RANDOM_PAGES_EVICT_NUM} here

Reply:
`RandomLruPageReplacementPolicy#RANDOM_PAGES_EVICT_NUM` is in internal package. Generated JavaDocs for AI release will be broken.